src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -194,6 +197,7 @@ public List<BucketMetadata> list(
         kind = kind.filter(s -> !s.isEmpty());
         contentType = contentType.filter(s -> !s.isEmpty());
         objectName = objectName.filter(s -> !s.isEmpty());
+        bucketName.filter(s -> !s.isEmpty());
```

Comment:
you need to write

bucketName = bucketName.filter(s -> !s.isEmpty());

Reply:
The bucketName should be declared as `final` to avoid issues in the lambda.
I'll create a new final temp variable for this